.travis.yml

```diff
@@ -22,24 +22,17 @@ matrix:
       sudo: required
       compiler: clang
     - os: osx
-      osx_image: xcode9
+      osx_image: xcode10.2
 
-#
-# Some of the OS X images don't have cmake, contrary to what people
-# on the Internet say
-#
 before_install:
```

Comment:
why keep the empty step?

Reply:
A mistake from begin on the flow, becomes removed.